packages/@aws-cdk/aws-lambda/lib/event-source-mapping.ts

```diff
@@ -351,4 +362,10 @@ export enum StartingPosition {
    * always read the most recent data in the shard
    */
   LATEST = 'LATEST',
+
+  /**
+   * Start reading from a position defined by a time stamp.
+   * Only supported for Amazon Kinesis streams.
```

Comment:
what happens when this is set for non Kinesis streams?

Reply:
Updated the docs to say that an error will occur